docs/experiments-covid.md

```diff
@@ -24,18 +24,19 @@ We've kept around older versions of the index for archival purposes &mdash; scro
 
 ## Data Prep
 
-The latest distribution available is from 2020/04/24.
+The latest distribution available is from 2020/05/01.
 First, download the data:
 
 ```bash
-DATE=2020-04-24
+DATE=2020-05-01
 DATA_DIR=./cord19-"${DATE}"
 mkdir "${DATA_DIR}"
 
 wget https://ai2-semanticscholar-cord-19.s3-us-west-2.amazonaws.com/latest/comm_use_subset.tar.gz -P "${DATA_DIR}"
 wget https://ai2-semanticscholar-cord-19.s3-us-west-2.amazonaws.com/latest/noncomm_use_subset.tar.gz -P "${DATA_DIR}"
 wget https://ai2-semanticscholar-cord-19.s3-us-west-2.amazonaws.com/latest/custom_license.tar.gz -P "${DATA_DIR}"
 wget https://ai2-semanticscholar-cord-19.s3-us-west-2.amazonaws.com/latest/biorxiv_medrxiv.tar.gz -P "${DATA_DIR}"
+wget https://ai2-semanticscholar-cord-19.s3-us-west-2.amazonaws.com/latest/arxiv.tar.gz -P "${DATA_DIR}"
```

Comment:
Add a line for Arxiv below in the comments too?

Reply:
yup, good catch.